static/js/admin.js

```diff
@@ -438,6 +438,7 @@ function _setup_page() {
             },
             success: function () {
                 var button = row.find("button.deactivate");
+                row.find('button.open-user-form').hide();
                 button.addClass("btn-warning");
```

Comment:
I am not saying it is wrong. `row.find('.open-user-form').hide();` looks more clean. You can leave it as it is if it matches the style of the rest of the file. :)